JitsiConference.js

```diff
@@ -1636,6 +1653,18 @@ JitsiConference.prototype.onRemoteTrackAdded = function(track) {
         return;
     }
 
+    // Setup E2EE handling, if supported.
+    if (this._e2eeCtx) {
+        const activeTPC = this.getActivePeerConnection();
```

Comment:
is E2EE supposed to be used only with the JVB connection?

Reply:
Yes